app/src/main/java/net/programmierecke/radiodroid2/station/ItemAdapterStation.java

```diff
@@ -360,10 +360,13 @@ public void onClick(View view) {
             holder.stubDetails = null;
             holder.viewTags = (TagsView) holder.viewDetails.findViewById(R.id.viewTags);
             holder.buttonStationLinks = holder.viewDetails.findViewById(R.id.buttonStationWebLink);
+            holder.buttonStationLinks.setImageResource(R.drawable.ic_store_black_24dp);
```

Comment:
But it is already defined in xml:
```
        <ImageButton
                android:id="@+id/buttonStationWebLink"
                style="@style/IconButton"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:contentDescription="@string/detail_weblink"
                android:background="?attr/selectableItemBackgroundBorderless"
                android:tint="?attr/iconsInItemBackgroundColor"
                app:srcCompat="@drawable/ic_store_black_24dp"/>
```

Why we need to set it from code?

Reply:
I haven't found the answer yet. Next time I'll update the libraries one by one. Fortunately I did this with okhttp.